src/e2e-test/java/io/cdap/plugin/S3Client.java

```diff
@@ -0,0 +1,78 @@
+package io.cdap.plugin;
```

Comment:
all files should have a copyright header, similar comment for the other files in this PR. Checkstyle is supposed to fail if there isn't a copyright, do we not have that setup for this repo?

Reply:
I have run the ` mvn clean package -DskipTests` locally and no test failure is reported. I will add the `CopyRight` header anyway.